packages/lab/src/stepper-input/StepperInput.tsx

```diff
@@ -51,94 +98,48 @@ export const StepperInput = forwardRef<HTMLDivElement, StepperInputProps>(
       window: targetWindow,
     });
 
-    const adornmentRef = useRef<HTMLDivElement | null>(null);
     const inputRef = useRef<HTMLInputElement | null>(null);
 
     const {
-      decrementButtonDown,
-      getButtonIcon,
       getButtonProps,
       getInputProps,
-      incrementButtonDown,
       isAtMax,
       isAtMin,
-      refreshCurrentValue,
       stepperDirection,
-      valuesHaveDiverged,
     } = useStepperInput(props, inputRef);
 
-    useActivationIndicatorPosition(
-      adornmentRef,
-      valuesHaveDiverged() || showRefreshButton
-    );
-
     const endAdornment: ReactNode = (
-      <div className={withBaseName("adornmentContainer")} ref={adornmentRef}>
+      <div className={withBaseName("buttonContainer")}>
+        <Button
+          className={withBaseName("stepperButton")}
+          disabled={isAtMax()}
+          {...getButtonProps(stepperDirection.INCREMENT, ButtonPropsProp)}
+        >
+          <TriangleUpIcon aria-label="triangle-up-icon" />
+        </Button>
         <Button
-          aria-label="Refresh default value"
-          className={clsx(withBaseName("secondaryButton"), {
-            // Refresh button is always rendered and has its visibility toggled to
-            // avoid component width changing.
-            [withBaseName("hideSecondaryButton")]: !(
-              showRefreshButton || valuesHaveDiverged()
-            ),
-          })}
-          onClick={refreshCurrentValue}
-          variant="secondary"
+          className={withBaseName("stepperButton")}
+          disabled={isAtMin()}
+          {...getButtonProps(stepperDirection.DECREMENT, ButtonPropsProp)}
         >
-          <RefreshIcon aria-label="refresh" />
+          <TriangleDownIcon aria-label="triangle-down-icon" />
```

Comment:
Icons should have aria-hidden and the button should have aria-label

Reply:
thanks for the comments, made all the changes